### PR TITLE
Use integer column type and unsigned modifier

### PR DESCRIPTION
**Pull request by carusogabriel**

### CODE REVIEW

**Comment by JosephSilber** [comment]
How is this any better?

File: migrations/create_bouncer_tables.php [hunk 1]
```diff
@@ -56,7 +56,7 @@ public function up()
         });
 
         Schema::create(Models::table('permissions'), function (Blueprint $table) {
-            $table->integer('ability_id')->unsigned()->index();
+            $table->unsignedInteger('ability_id')->index();
             $table->unsignedInteger('entity_id');
```
Comment (carusogabriel) [inline migrations/create_bouncer_tables.php]: @JosephSilber As we use once, why not use and standardize? :smile:

Comment (JosephSilber) [reply migrations/create_bouncer_tables.php]: I find single-word method to be more legible.

Comment (JosephSilber) [reply migrations/create_bouncer_tables.php]: If you want to to you can change the PR the other way around: change all `unsignedInteger`s to `integer(...)->unsigned()`.

**Pull request merged**